Animations for Board and Hole Cards

VonIobro opened the pull request:
Issue https://github.com/acebusters/ab-web/issues/780
* seperate board from table component
* add boardcard-entry animation

Animations checked on 
- [x] firefox
- [x] chrome

sunify commented:
I can't see my cards (and opponent's cards in showdown) in firefox

![image](https://user-images.githubusercontent.com/1477583/32370752-326e358e-c0c9-11e7-9322-4778d00c49c4.png)

VonIobro commented:
@sunify Ready to merge.
* I simplified the staggered anim, as the original was slowing down my computer? Anways works great now.
* I removed the HoleCards anim, and will work on that as a seperate PR
